pkg/api/clients/pubsub/pubsub.gen.go

```diff
@@ -0,0 +1,376 @@
+// Package statestore provides primitives to interact with the openapi HTTP API.
```

Comment:
Would it also be better to have it under a folder `generated` ? For all generated clients?

Reply:
@artursouza @mukundansundar  Question: Why do we want to put the open API generated clients into dapr/dapr at all? Putting the clients in this repo means that:
1. We will have schema drift and duplication between the repos. We don't do this for kit or components-contrib, so I don't think we should be doing it for the open API schema either. Changing the Dapr runtime API should be a very intentional task, and getting a signal something has changed in the dapr runtime API because the generated client tests are failing is a good thing.
2. Any consumers which want to use the open API clients are going to have to also import dapr/dapr, which brings with it dependency bloat and potential incompatibilities for that consuming project. We should treat  basically all of dapr/dapr as inside a go `/internal` package imo.

I think we should have a clear separation of the clients to the main dapr/dapr repo. This also goes for the `pkg/apis` packages, but that is a separate discussion which I would like to address in future.